### PR TITLE
Display list of jobs in error message for schedule job

### PR DESCRIPTION
**Pull request by vritant**
@nguyenfilip  please take a look and merge if you like it, im hoping I can mention this in demo tomorrow

### CODE REVIEW

File: server/src/main/java/org/candlepin/resource/JobResource.java [hunk 1]
```diff
@@ -220,6 +222,15 @@ public JobStatus schedule(@PathParam("task") String task) {
         }
     }
 
+    private String prettyPrintJobs(String... jobs) {
```
Comment (nguyenfilip) [inline server/src/main/java/org/candlepin/resource/JobResource.java]: What do you think about using StringUtils instead?

```java
        List<String> jobNames = new ArrayList<String>();
        for (String job : jobs) {
            jobNames.add(StringUtils.substringAfterLast(job, "."));
        }
        return StringUtils.join(jobNames,", ");
```

**Comment by nguyenfilip** [comment]
@vritant This is good as is. Let me know if you want to use StringUtils.

**Comment by vritant** [comment]
@nguyenfilip hold please. I just realized I did not add @ApiOperation

File: server/src/main/java/org/candlepin/resource/JobResource.java [hunk 2]
```diff
@@ -207,7 +208,9 @@ public JobStatus schedule(@PathParam("task") String task) {
                 return pk.scheduleSingleJob((Class<? extends KingpinJob>) taskClass, Util.generateUUID());
             }
             else {
-                throw new BadRequestException(i18n.tr("Not a permissible job: {0}", task));
+                throw new BadRequestException(i18n.tr(
+                    "Not a permissible job: {0}. Only {1} are permissible", task,
+                    prettyPrintJobs(ConfigProperties.DEFAULT_TASK_LIST)));
```
Comment (Ceiu) [inline server/src/main/java/org/candlepin/resource/JobResource.java]: :+1:

**Pull request merged**